### PR TITLE
Release/1.1.x

### PR DESCRIPTION
**Pull request by AndyKIron**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,162 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.1.0-rc.21](https://github.com/ironSource/fusion-ui/compare/v1.1.0-rc.20...v1.1.0-rc.21) (2022-02-23)
```
Comment (danielbrazg) [inline CHANGELOG.md:5]: remove all rc.X mentions. instead add everything under version 1.1.0

File: projects/fusion-ui/src/lib/components/ui-components/dropdown/dropdown/dropdown.component.ts [hunk 2]
```diff
@@ -457,7 +458,11 @@ export class DropdownComponent extends StyleBase implements OnInit, OnDestroy, O
                 (!this.autoComplete && !this.backendPaginationState && this.searchValue && this.searchValue.value) ||
                 this.isPredefinedTags
             ) {
-                const filterByArray: string[] = this.mappingOptions ? Object.values(this.mappingOptions) : [];
+                const filterByArray: string[] = this.searchByProperties?.length
```
Comment (danielbrazg) [inline projects/fusion-ui/src/lib/components/ui-components/dropdown/dropdown/dropdown.component.ts:461]: this is not readable. please change to if-else conditions

**Review by danielbrazg — APPROVED** [review]

File: projects/fusion-ui/src/lib/components/ui-components/dropdown/dropdown/dropdown.component.ts (hunk 2 above)
Comment (AndyKIron) [reply projects/fusion-ui/src/lib/components/ui-components/dropdown/dropdown/dropdown.component.ts:461]: Ok.

File: CHANGELOG.md (hunk 1 above)
Comment (AndyKIron) [reply CHANGELOG.md:5]: will do it after or right before the release build I think.

**Pull request merged**